target/linux/generic/files/drivers/platform/mikrotik/rb_hardconfig.c

```diff
@@ -562,11 +588,13 @@ static int hc_wlan_data_unpack(const u16 tag_id, const size_t tofs, size_t tlen,
 
 	ret = -ENODATA;
 	switch (magic) {
+	case RB_MAGIC_LZ77:
+		fallthrough;
```

Comment:
`fallthrough` unnecessary here:

```c
switch (magic) {
	case RB_MAGIC_LZ77:
	case RB_MAGIC_LZOR:
	...
```

(`fallthrough` is only necessary to avoid compiler warning when you have active code in a switch case that falls through another case).